README.md

```diff
@@ -23,27 +23,119 @@ https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/ (OS X)
 
 https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/ (Windows)
 
-
 Then, start up the database using:
 
 ```
 $ mongod
 ```
 
-
 With `haiku-data.json` in your current working directory run this command to import the data:
 
 ```
 $ mongoimport --db haiku --collection users --drop --file haiku-data.json
 ```
 
 
+## Mongo CRUD Operations
+
+First, start up the mongo shell:
+
+```
+$ mongo
+```
+
+To view all databases:
+
+```
+> show dbs
+```
+
+Use specific database:
+
+```
+> use `db_name`
+```
+
+View collections of current database:
+
+```
+> show collections
+```
+
+View contents of current database:
+
+```
+> db.your_collection.find().pretty()
+```
+
+Create operations to add new documents to a collection:
+
+```
+> db.collection.insert()
+> db.collection.insertOne()
+> db.collection.insertMany()
+```
+
+Read operations to retrieve documents from a collection:
+
+```
+> db.collection.find()
```

Comment:
Can you add an example of a predicate here - like show all users whose status value is 'on'?